mysql-test/suite/sys_vars/t/thread_cache_size_func.test

```diff
@@ -75,12 +75,10 @@ DISCONNECT conn4;
 # Checking the status
 #
 
-# Wait until all disconnects ready
-let $wait_condition= SELECT COUNT(*)= 1 FROM INFORMATION_SCHEMA.PROCESSLIST;
---source include/wait_condition.inc
-
-SHOW STATUS LIKE 'Threads_cached';
---echo 3 Expected
+--echo # Threads_cached must become 3
+--let $status_var= Threads_cached
+--let $status_var_value= 3
+--source include/wait_for_status_var.inc
 
```

Comment:
After the Threads_cached variable becomes 3 - it could be useful to check if SELECT COUNT(*)= 1 FROM INFORMATION_SCHEMA.PROCESSLIST;